lib/web/apps.go

```diff
@@ -73,7 +73,7 @@ func (h *Handler) clusterAppsGet(w http.ResponseWriter, r *http.Request, p httpr
 			Apps:              apps,
 		}),
 		StartKey:   resp.NextKey,
-		TotalCount: len(apps),
+		TotalCount: resp.TotalCount,
```

Comment:
did the previous cause a bug? if you look at line 65 `removeUnsupportedApps`, this is the true count of the apps

Reply:
i don't think this change is correct: https://github.com/gravitational/teleport/commit/dd67a49ab5035e51847a3fc8934f53b5abe25a6a#diff-b14891674f8dd044033299cfe41f2bad8bd692e2661118d685c612523f37f264L77

thinking on it, i think the prior change is incorrect too, b/c the next page can result in some varying numbers... which will keep changing the total count: https://github.com/gravitational/teleport/pull/18586/files#diff-b14891674f8dd044033299cfe41f2bad8bd692e2661118d685c612523f37f264

do you mind adding some dummy apps with supported and unsupported and test that theory ^. i think the proper way to handle this is to add a filter option to filter out unsupported apps

Reply:
I have created an issue and a separate PR to address this: https://github.com/gravitational/teleport/pull/22232